### ISSUE TITLE
Please fix conflict of types

### ISSUE THREAD

**Issue by pizza-777** [broxus/everscale-standalone-client#4]
Code:
```js
const ever = new ProviderRpcClient({ 
      fallback: () => 
        EverscaleStandaloneClient.create({
          connection: 'mainnet',
        }),
    });
```

Error:
```error
(property) fallback?: (() => Promise<Provider>) | undefined
Provider factory which will be called if injected provider was not found. Can be used for initialization of the standalone Everscale client
Type '() => Promise<EverscaleStandaloneClient>' is not assignable to type '() => Promise<Provider>'.
  Type 'Promise<EverscaleStandaloneClient>' is not assignable to type 'Promise<Provider>'.
    Type 'EverscaleStandaloneClient' is missing the following properties from type 'Provider': addListener, removeListener, on, once, and 2 more.ts(2322)
```

**Comment by Rexagon** [comment]
Thanks for the issue! It seems that there is a problem with VSCode, it can't resolve type which implements these methods indirectly.

Fixed with f167a6e6, please update both `everscale-inpage-provider` (0.3.33) and `everscale-standalone-client` (2.0.26)

**Comment by pizza-777** [comment]
Thank You
